scripts/tests/refresh-flux-ghcr-auth/fake_kubectl_test.go

```
@@ -0,0 +1,871 @@
+	revision := os.Getenv("EXPECTED_GHCR_REVISION")
+	image := os.Getenv("EXPECTED_KSAIL_TARGET_IMAGE")
+	verifiedImage := defaultString(os.Getenv("FAKE_TALOS_VERIFIED_IMAGE"), image)
+	nodes := []any{
+		fakeInventoryNode("prod-worker-1", "prod-worker-1-uid", "10.0.0.2", "198.51.100.2", false, revision, "", "", true),
+		fakeInventoryNode("prod-control-plane-1", "prod-control-plane-1-uid", "10.0.0.1", "198.51.100.1", true, revision, "", "", true),
+		fakeInventoryNode("prod-control-plane-2", "prod-control-plane-2-uid", "10.0.0.3", "198.51.100.3", true, revision, revision, image, false),
+		fakeInventoryNode("prod-control-plane-3", "prod-control-plane-3-uid", "10.0.0.4", "198.51.100.4", true, revision, revision, image, false),
+	}
+	if os.Getenv("FAKE_ALL_TALOS_NODES_STALE") == "true" {
+		for _, node := range nodes {
+			nodeMap := node.(map[string]any)
```

golangci-lint (2.12.2)

[error] type assertion must be checked

(forcetypeassert)